app/helpers/application_helper/page_layouts.rb

```diff
@@ -58,6 +58,8 @@ def layout_uses_listnav?
 
     return false if controller.action_name.end_with?("tagging_edit")
 
+    return false if controller.kind_of?(GenericObjectDefinitionController) && x_node != "root"
```

Comment:
@ZitaNemeckova this line seems to have no effect, not sure why this is needed.

Reply:
@h-kataria It's needed for dislaying summary of a Generic Object Definition as it uses `show_list` instead of `show`.

With the line:
<img width="941" alt="Screenshot 2020-06-01 at 10 52 39" src="https://user-images.githubusercontent.com/9210860/83392969-4bf74500-a3f6-11ea-8922-432c1647166d.png">

Without the line:
<img width="1123" alt="Screenshot 2020-06-01 at 10 54 22" src="https://user-images.githubusercontent.com/9210860/83392965-4863be00-a3f6-11ea-857d-09a804d28ebf.png">

Reply:
If the line stays I'll make a follow up PR to fix toolbar.